src/test/java/com/commercetools/sync/customers/utils/StoreUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,323 @@
+package com.commercetools.sync.customers.utils;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.customers.Customer;
+import io.sphere.sdk.customers.CustomerDraft;
+import io.sphere.sdk.customers.CustomerDraftBuilder;
+import io.sphere.sdk.customers.commands.updateactions.AddStore;
+import io.sphere.sdk.customers.commands.updateactions.RemoveStore;
+import io.sphere.sdk.customers.commands.updateactions.SetStores;
+import io.sphere.sdk.models.KeyReference;
+import io.sphere.sdk.models.ResourceIdentifier;
+import io.sphere.sdk.stores.Store;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static com.commercetools.sync.customers.utils.CustomerUpdateActionUtils.buildStoreUpdateActions;
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class StoreUpdateActionUtilsTest {
+
+    @Test
+    void buildStoreUpdateActions_WithSameStores_ShouldNotReturnAction() {
+        final Customer oldCustomer = mock(Customer.class);
```

Comment:
Can declare it as global variable, and initialise it by `beforeEach` annotation.

Reply:
It's better to duplicate in tests, otherwise, it's hard to read and understand a scenario.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/591/commits/83347ed4128a58d66e233f95c8b8f535b6985ead